pandas/tests/groupby/test_function.py

```diff
@@ -1507,14 +1507,22 @@ def test_quantile_missing_group_values_no_segfaults():
         grp.quantile()
 
 
-def test_quantile_missing_group_values_correct_results():
-    # GH 28662
-    data = np.array([1.0, np.nan, 3.0, np.nan])
-    df = pd.DataFrame(dict(key=data, val=range(4)))
+@pytest.mark.parametrize(
+    "key, val, expected_key, expected_val",
+    [
+        ([1.0, np.nan, 3.0, np.nan], range(4), [1.0, 3.0], [0.0, 1.0]),
```

Comment:
Is this expected output correct? I would think expected_val should be [0.0, 2.0]

Reply:
The original version was like that, but I think the same, it should be `[0.0, 2.0]`,